strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/StrongboxAuthenticationFilter.java

```diff
@@ -8,32 +8,32 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import org.carlspring.strongbox.security.authentication.suppliers.AuthenticationSuppliers;
+import org.carlspring.strongbox.security.authentication.strategy.DelegatingAuthenticationConverter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.security.authentication.AuthenticationManager;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.context.SecurityContextHolder;
-import org.springframework.web.filter.OncePerRequestFilter;
+import org.springframework.security.web.authentication.AuthenticationFilter;
 
 /**
  * @author Przemyslaw Fusik
  */
 public class StrongboxAuthenticationFilter
```

Comment:
@soapdogg you shouldn't override `doFilterInternal` method here. The point of this task was to use original `AuthenticationFilter` implementation to work same way as it was before with `StrongboxAuthenticationFilter`.

Reply:
Make `org.carlspring.strongbox.security.authentication.StrongboxAuthenticationFilter extend the org.springframework.security.web.authentication.AuthenticationFilter` is in the issue description. 

It seems like what you are saying an this is contradicting each other. Which implementation is preferred?

Reply:
I mean that you can extend but shouldn't just override everything, obviously it doesn't make sense. 
sorry if it was not clear :)